blog/2023-06-23-newsletter.mdx

```diff
@@ -0,0 +1,164 @@
+
+<p>
+    <i>
+        <small>
+            Image credit: <a href="https://unsplash.com/photos/sN40hKv8_cI">Road Trip with Raj on Unsplash</a>
+        </small>
+    </i>
+</p>
+
+**Welcome to the first ever conda community newsletter!** The newsletter will highlight recent and upcoming activity that is relevant to the [conda community](https://conda.org/). This first issue highlights releases, grants, events, ....
+
+We are hoping these will become a semi-regular feature in the conda landscape going forward. If you have an item to include in a future newsletter, please let us know by posting it to the [Conda Communications Matrix Channel](https://matrix.to/#/%23conda-communications:matrix.org) or by creating an issue in the [Conda Communications GitHub Repo](https://github.com/conda/communications/issues).
+
+<!-- truncate -->
+
+Finally, if you want to receive these newsletters from the comfort of your inbox, **please sign up for the [Conda Announce Mailing list](https://lists.conda.org/wws/info/announce).**
+
+Happy reading,<br />
+The Editors
+
+---
+
+## Community Gathering Spaces
+
+It has been a busy year for creating and consolidating online gathering spaces for the conda community. We now have a community web site, a Q&A/forum stie, a unified chat platform, a mailing list, and more. See **[Pinak Datta's recent post] (/blog/2023-06-22-conda-spaces-blogpost)** for a complete list and links.
+
+### Community calls
+
+Please join us every other week to learn what is going on in the conda community and to provide your input.
+
+* The Mamba project has [bi-weekly community calls](https://hackmd.io/iOgkxNMpTea6lWczjCov9Q?view) on Tuesdays. Our next call is June 27.
+* The overall conda community has [bi-weekly calls](https://hackmd.io/@conda-community?tags=%5B%22meeting-notes%22%5D) on Wednesdays. The next one is on July 5.
+
+---
+
+## Upcoming events
+
+[<Image img={scipybanner} alt="Banner image for SciPy2023"/>](https://www.scipy2023.scipy.org/)
+
+### SciPy 2023 Conda & Friends Sprint, July 15-16, Austin
+
+***Probably...*** *We have proposed a sprint for SciPy, but we haven't heard back. In the past the conference has approved our requests. So, until that happens, we are *tentatively* announcing that there will be a Conda & Friends sprint at SciPy, but we reserve the right to take it back if our foresight skills are weak.*
+
+> The conda community will again be meeting at [SciPy 2023](https://www.scipy2023.scipy.org/) in Austin, Texas for two days of collaborative work.
+>
+> [Sprints](https://www.scipy2023.scipy.org/sprints) are open to everyone who is interested in contributing to the various projects in the conda ecosystem (e.g. projects under the [conda](https://github.com/conda), [conda-incubator](https://github.com/conda-incubator), [mamba-org](https://github.com/mamba-org), [conda-forge](https://github.com/conda-forge), ... GitHub organizations), what we call "conda and friends".
```

Comment:
And I presume this is waiting for something, too?

Reply:
No, @chenghlee has still not heard back.  My instinct is to publish this with the "probably" disclaimer.

Reply:
Leaving this in, even though it is only "probable."